scripts/dangerjs/detectNonApprovedDependencies/approvedPackages.ts

```diff
@@ -84,6 +84,7 @@ export default [
   '@popperjs/core@2.1.1',
   '@popperjs/core@2.4.0',
   '@popperjs/core@2.4.2',
+  'popper-max-size-modifier@0.2.0',
```

Comment:
Has friendly MIT license 🐱

Reply:
Great that you checked it, cause I did not...